cmd/detect/main.go

```diff
@@ -31,6 +34,16 @@ func main() {
 // TODO: implement the following
 // 		- install nodjs, npm, and yarn if needed
 func runDetect(context detect.Detect) (int, error) {
+	// TODO: include to image of cloudfoundry:cnb
+	_, err := exec.LookPath("bundle")
+	if err != nil {
+		_, err := exec.Command("gem", "install", "bundler", "-v", "1.17.3", "--user-install").CombinedOutput()
```

Comment:
What happens in the case that this environment is not connected to the internet? Wouldn't this just always fail?